src/dmd/arrayop.d

```diff
@@ -13,6 +13,7 @@
 
 module dmd.arrayop;
 
+// import core.stdc.stdio : ;
 import core.stdc.stdio;
```

Comment:
What I mean is if `core.stdc.stdio` is unused in the given file this diff should read 
```diff
-import core.stdc.stdio;
```

Reply:
I misinterpreted that the stdio was programmer preference rather than caused build failures and I flubbed up some of the patch. :/

But the more I dig into imports and the fragility of the code base, I'm not familiar enough with the nuances of D semantics to be effective here.  I'll delete this fork in a couple of days and would recommend the patch was reverted in master if it hasn't already been done.  I don't understand the implications of changing imports and the compiler isn't strict enough to protect me here either.

Some of the issues that I found that I think should be addressed before import modification:
1)  Making an import selective and/or local can double the compile time or significantly increase it
2)  Reordering imports causes symbols to not be public in other modules.
3)  Reordering imports can cause template recursion errors
4)  Apparently there are linking issues, but I did not observe these.

I did observe localizing imports for a few key modules did decrease compile time by 10-20%, so seem like it's worth doing.

Thank you for your time and sorry for the distractions.